dev-docs/bidders/colossus.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: bidder
+title: Colossus
+description: Prebid Colossus Bidder Adaptor
+pbjs: true
```

Comment:
This looks like a copy / paste error, as `pbjs` is defined correctly as false on line 12.